elm-analyse.json

```diff
@@ -0,0 +1,8 @@
+{
+    "excludedPaths": [
+        "src/Scale",
+        "src/Axis.elm",
+        "src/Force.elm",
```

Comment:
In Axis.elm `computeOptions`, `horizontalAttrs`, `verticalAttrs` had verbose type signatures.
Force.elm had unused warnings for types `Collision`, `X`, `Y` probably related to `--Debug.crash "not implemented"`.

I might submit an issue to elm-analyse to ask for more granular configuration options, so we can dismiss single rules for a file instead of entire files.

Reply:
[Issue](https://github.com/stil4m/elm-analyse/issues/244) is created.